File: Languages/Day11/ans.rs
// Question 1

fn main() {
    let favorite_foods = vec!["apple", "papaya", "dragonfruit"];
    println!("{:?}", favorite_foods);
    println!("{}", favorite_foods[0]);
    println!("{}", favorite_foods[2]);
}

//Question 2

fn main() {
    let user_profile = ("Josh", 22, false);
    println!("{:?}", user_profile);
}

//Question 3

#[derive(Debug)]
struct Book {
    title: String,
    author: String,
    pages: u32,
}

fn main() {
    let book = Book {
        title: String::from("Apple"),
        author: String::from("Steve Jobs"),
        pages: 549,
    };

    println!("{:?}", book);
    println!("{}", book.title);
}

//Question 4

use std::collections::HashSet;

fn main() {
    let skills = HashSet::from([
        "boxing",
        "Muay Thai",
        "karate",
        "Muay Thai",
        "kickboxing",
    ]);

    println!("{:?}", skills);
    println!("{}", skills.contains("boxing"));
}

//Question 5
#[derive(Debug)]
struct Student {
    name: String,
    languages: Vec<String>,
    active: bool,
}

fn main() {
    let student = Student {
        name: String::from("Mike"),
        languages: vec![
            String::from("English"),
            String::from("Spanish"),
            String::from("French"),
        ],
        active: true,
    };

    println!("{:?}", student);
    println!("{:?}", student.languages);
}

File: Languages/Week 2/Day08/ans.py
#question 1
from oauthlib.uri_validate import authority

favorite_foods = ['apple', 'papaya', 'dragonfruit']

print(favorite_foods)
print(favorite_foods[0])
print(favorite_foods[2])
favorite_foods.append('grapes')
print(len(favorite_foods))

#question 2

user_profile = ('bob', 36, True)
print(user_profile)

#question 3

book = {
    "title": "bobbie did it again",
    "author": "ron johnson",
    "pages": 465,
}
print(book)
print(book['title'])

#question 4

skills = {"boxing", "Muay Thai", "karate", "kickboxing", "Muay Thai"}
print(skills)
print("MMA" in skills)

#question 5

student = {
    "name" : "bobby",
    "languages" : ["English", "Spanish", "British English"],
    "active" : True
}
print(student)
print(student.get('languages'))